staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -687,6 +689,29 @@ message LinuxContainerConfig {
     LinuxContainerSecurityContext security_context = 2;
 }
 
+// WindowsSandboxSecurityContext holds platform-specific configurations that will be 
+// applied to a sandbox.
+// These settings will only apply to the sandbox container.
+message WindowsSandboxSecurityContext {
+    // User name to run the container process as. If specified, the user MUST
+    // exist in the container image and be resolved there by the runtime;
+    // otherwise, the runtime MUST return error.
+    string run_as_username = 1;
+
+    // The contents of the GMSA credential spec to use to run this container.
+    string credential_spec = 2;
```

Comment:
this is surprising to me... was this missing from the v1 CRI API?

Reply:
should this avoid mixing fixes or other additions into this PR, and keep this focused on the hostProcess addition?

Reply:
These updates were discussed in the hostProcess KEP but I can split them out into a separate commit or PR if that would be easier to track.

https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/1981-windows-privileged-container-support#cri-implementation-details